src/app/interfaces/events/ha-status-event.interface.ts

```diff
@@ -1,6 +1,7 @@
 import { FailoverDisabledReason } from 'app/enums/failover-disabled-reason.enum';
+import { HaStatusText } from 'app/enums/ha-status-text.enum';
 
 export interface HaStatus {
-  status: 'HA Enabled' | 'HA Disabled';
+  status: HaStatusText;
```

Comment:
Can't we just have a boolean flag here? 😭

Reply:
@undsoft 
I've just refactored, not sure about this status property :)

Reply:
Please add a TODO line to make this a boolean variable.